src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(Color color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    public double getArea() {
+        return side * side;
+    }
+
+    public String draw() {
+        return "Figure: Square, Area: "
+                + getArea()
+                + " units, side: "
+                + side
+                + " units, "
+                + " Figure color: "
+                + getColor();
+    }
```

Comment:
The Square class should not extend Figure directly if Figure is intended to be an abstract class or interface according to the checklist. Consider whether Figure should be an abstract class or an interface based on the provided behavior, and use it appropriately.